config/routes.rb

```
@@ -193,9 +194,15 @@
           post :add_participant
           delete :delete_participants
         end
+      end      
+      resources :reports, only: [] do
+        collection do
+          post :fetch_response_report
+        end
       end
```

RuboCop (1.86.2)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)